### ISSUE TITLE
Imported fragment not recognized

### ISSUE THREAD

**Issue by tjwelde** [kamilkisiela/graphql-inspector#729]
Given a fragment:
```
import gql from 'graphql-tag';

export default gql`
  fragment Teaser on NodeArticle {
    title
    created
  }
`;
```

and a query:
```
import gql from 'graphql-tag';

import TEASER_FRAGMENT from './fragment';

export default gql`
  query node($id: String!) {
    nodeById(id: $id) {
      ...Teaser
    }
  }
  ${TEASER_FRAGMENT}
`;
```

The command
```
graphql-inspector validate ./test/node.query.js ./test/schema.graphql
```

returns
```
error Node does not exist: Teaser
```

If I copy the fragment into the `node.query.js` file, either directly in the query, or as a variable, it works.

Do you have any idea, why that happens? I tried with node v10 and node v12

**Comment by kamilkisiela** [comment]
We don't use AST to look for variables and resolve them but just to find template literals. The logic behind this drops variables so you have to define a pattern that includes both files (fragment and query).

Something like:

    graphql-inspector validate './test/*.query.js' ./test/schema.graphql

**Comment by tjwelde** [comment]
Hey @kamilkisiela,

ah, thanks for the clarification, I already got the feeling, that you do something like that, after I investigated the issue more. 
Also good, that you show the DOCUMENTS argument in your command in quotes, because I had another issue with glob patterns, because my `fish` shell apparently always expanded these glob patterns into multiple arguments.

Now that everything is clarified, it works perfectly.

I would propose adding these informations to the documentation. e.g. here: https://github.com/kamilkisiela/graphql-inspector/pull/733
